src/market.py

```diff
@@ -60,7 +58,7 @@ def load(
         endpoint: str, market_address: str, options: Any, program_id: PublicKey = DEFAULT_DEX_PROGRAM_ID
```

Comment:
I think we should abstract away these `endpoint` and `program_id` params by adding an init function to market.